trac-env/conf/trac.ini

```diff
@@ -139,8 +139,8 @@ name = Django
 url = https://code.djangoproject.com/
 
 [query]
-default_anonymous_query = status!=closed&desc=1&order=id
-default_query = status!=closed&desc=1&order=id
+default_anonymous_query = description~=&status!=closed&desc=1&order=id&col=id&col=summary&col=type&col=owner&col=component
+default_query = description~=&status!=closed&desc=1&order=id&col=id&col=summary&col=type&col=owner&col=component
```

Comment:
Love this 🤩 
Could you extract this to a separate commit (or even a separate PR so I can merge/deploy it right away)?

Reply:
https://github.com/django/code.djangoproject.com/pull/211 :+1: